lib/job.js

```diff
@@ -224,10 +224,12 @@ class Job extends BaseModel {
         const newJob = await super.update();
         const pipeline = await pipelineFactory.get(newJob.pipelineId);
 
+        const isDisabled = this.state === 'DISABLED';
+
         try {
             const newPeriodic = getAnnotations(this.permutations[0], 'screwdriver.cd/buildPeriodically');
 
-            if (newPeriodic && newPeriodic !== oldPeriodic) {
+            if (newPeriodic && newPeriodic !== oldPeriodic && !isDisabled) {
```

Comment:
need to add a check to detect  transition from old state disable -> new state enabled, else jobs with disabled -> enabled state transition won't have periodic build configuration